src/components/v2/V2Create/forms/TokenForm/index.tsx

```diff
@@ -144,6 +176,23 @@ export default function TokenForm({ onFinish }: { onFinish: VoidFunction }) {
     redemptionRate,
   ])
 
+  const defaultValueStyle: CSSProperties = {
+    color: colors.text.tertiary,
+    marginLeft: 15,
+  }
+
+  const reservedRatePercent = parseFloat(
+    formatReservedRate(BigNumber.from(reservedRate)),
+  )
+
+  const discountRatePercent = parseFloat(
+    formatDiscountRate(BigNumber.from(discountRate)),
+  )
+
+  // Tokens received by contributor's per ETH
+  const initialIssuanceRate =
+    DEFAULT_ISSUANCE_RATE - reservedRatePercent * 10000
```

Comment:
can use `reservedRateFrom(reservedRatePercent)` here i thnk

Reply:
Edit: nah, should be

```suggestion
    DEFAULT_ISSUANCE_RATE - reservedRatePercent * MAX_RESERVED_RATE
```